### ISSUE TITLE
Separate data loading code out into new git "nowcasting_data_loader" repo

### ISSUE THREAD

**Issue by JackKelly** [openclimatefix/predict_pv_yield_2#29]
After #16

**Comment by JackKelly** [comment]
Started work on new library: https://github.com/openclimatefix/nowcasting_dataloader

**Comment by JackKelly** [comment]
This is really a duplicate of #16
